(dp1
S'output'
p2
S"<type 'exceptions.TypeError'> <gluon.validators.IS_NOT_EMPTY object at 0x7ffb14674b10> is not JSON serializable"
p3
sS'layer'
p4
S'/home/matt/Dev/web2py/applications/blog/controllers/invite_tool.py'
p5
sS'code'
p6
S'\ndef index():\n    return locals()\t\n\t\ndef getAccountCollection():\n\treturn response.json(db().select(db.accounts.ALL, orderby=db.accounts.name))\n\t\ndef getInviteeCollection():\t\n\treturn response.json(db.invitees.account_id.contains(request.args)).select(orderby=db.invitees.first_name)\n\nresponse._vars=response._caller(getInviteeCollection)\n'
p7
sS'snapshot'
p8
(dp9
sS'traceback'
p10
S'Traceback (most recent call last):\n  File "/home/matt/Dev/web2py/gluon/restricted.py", line 227, in restricted\n    exec ccode in environment\n  File "/home/matt/Dev/web2py/applications/blog/controllers/invite_tool.py", line 11, in <module>\n  File "/home/matt/Dev/web2py/gluon/globals.py", line 412, in <lambda>\n    self._caller = lambda f: f()\n  File "/home/matt/Dev/web2py/applications/blog/controllers/invite_tool.py", line 9, in getInviteeCollection\n    return response.json(db.invitees.account_id.contains(request.args)).select(orderby=db.invitees.first_name)\n  File "/home/matt/Dev/web2py/gluon/globals.py", line 656, in json\n    return json(data, default=default or custom_json)\n  File "/home/matt/Dev/web2py/gluon/serializers.py", line 137, in json\n    value = json_parser.dumps(value, default=default)\n  File "/usr/lib/python2.7/json/__init__.py", line 250, in dumps\n    sort_keys=sort_keys, **kw).encode(obj)\n  File "/usr/lib/python2.7/json/encoder.py", line 207, in encode\n    chunks = self.iterencode(o, _one_shot=True)\n  File "/usr/lib/python2.7/json/encoder.py", line 270, in iterencode\n    return _iterencode(o, 0)\n  File "/home/matt/Dev/web2py/gluon/serializers.py", line 110, in custom_json\n    raise TypeError(repr(o) + " is not JSON serializable")\nTypeError: <gluon.validators.IS_NOT_EMPTY object at 0x7ffb14674b10> is not JSON serializable\n'
p11
s.